src/tests/flyer.test.ts

```diff
@@ -263,3 +263,25 @@ describe('deleteFlyer tests', () => {
     expect(deleteFlyerResponse).toBeNull();
   });
 });
+
+describe('editFlyer tests', () => {
```

Comment:
Would be good to eventually have a test that runs through the flyer lifecycle for images since that's the only external factor in our code -- create a flyer with a dummy image, update the image and ensure the old one was deleted, and then delete the flyer and make sure the image was deleted

You could create a folder within tests to store the images

Reply:
Sounds good, you can feel free to merge this PR once you address my comment above about the flyer ID -- you can implement this test later as well